app/kafka/manager/KafkaManager.scala

```diff
@@ -232,13 +232,38 @@ class KafkaManager(akkaConfig: Config) {
           Future.successful[ApiError \/ TopicIdentity](-\/(ApiError(s"Topic not found $topic for cluster $clusterName")))
         } { td =>
           tryWithKafkaManagerActor(KMClusterQueryRequest(clusterName, KSGetBrokers)) { brokerList: BrokerList =>
-            TopicIdentity.from(brokerList, td)
+            val metrics = getTopicMetrics(clusterName, topic)
+            TopicIdentity.from(brokerList, td, Some(metrics))
           }
         }
       })
     }
   }
 
+  def getTopicMetrics(clusterName: String, topic: String) : TopicMetrics = {
+    // TODO How to get the JMX host and port of the broker ?
```

Comment:
We should modify the cluster config to also allow for the input of JMX  configuration.  However, not everyone would use it so we should have a flag in the cluster config to turn JMX metrics on/off.  Based on this the cluster manager would then create or not create an actor which polls JMX metrics.